index.md

```diff
@@ -1,6 +1,61 @@
 ---
-# You don't need to edit this file, it's empty on purpose.
-# Edit theme's home layout instead if you wanna make some changes
-# See: https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-layout: home
+layout:     page
+title:      The Literature Review site of VITAL
+hidden:     true  # Excluded from the menu
 ---
+
+<style>
+    li {
+        margin-bottom: 0.7rem; /* This will make the lists easier to read. */
+    }
+</style>
+
+**Welcome!** Through this website, we will help each other stay up to date with new advances in our ever changing field. Our paper summaries will also help new members of VITALab to acquire essential knowledge about the state of the art. On this site, you'll find all the essential papers the VITALab members should be aware of.
```

Comment:
The goal of this website is for the VITALab member to stay up to date with new scientific publications.  This site also helps new members acquire knowledge on fundamental methods.